src/sql/magic_plot.py

```diff
@@ -103,12 +110,26 @@ def execute(self, line="", cell="", local_ns=None):
                 conn=None,
             )
         elif cmd.args.line[0] in {"hist", "histogram"}:
+            bin_specified = " --bins " in line or " -b " in line
```

Comment:
I tried `cmd.args.bins` but because of `bins` default value, `cmd.args.bins` always contains a value (either the default or user input) regardless of user's input.